rpc/client.go

```diff
@@ -18,6 +18,8 @@ var (
 	opRetries       = 0
 	opReadTimeout   = 15 * time.Second // client read
 	opWriteTimeout  = 15 * time.Second // client write
+	opSyncTimeout   = 30 * time.Second // client sync
+	opUnmapTimeout  = 30 * time.Second // client unmap
```

Comment:
What will happen if the io's are larger in size ? Sync and unmap may fail intermittently due to timeouts.